### PR TITLE
fixed to change 'species_only' parameter to 'column_filter' so we can pass any column to it

### PR DESCRIPTION
**Pull request by sweng66**

### CODE REVIEW

**Review by azurebrd — DISMISSED** [review]
oh, cool !

File: agr_literature_service/api/crud/topic_entity_tag_crud.py [hunk 1]
```diff
@@ -222,19 +222,27 @@ def show_source(db: Session, topic_entity_tag_source_id: int):
 def show_all_reference_tags(db: Session, curie_or_reference_id, token: str, page: int = 1,
                             page_size: int = None, count_only: bool = False,
                             sort_by: str = None, desc_sort: bool = False,
-                            species_only: bool = False, species: str = None):
+                            column_filter: str = None, species: str = None):
```
Comment (valearna) [inline agr_literature_service/api/crud/topic_entity_tag_crud.py:225]: The new name is a bit confusing. `column_filter` should be 'column_value_only` or something similar, and `species` should be `column_filter`

**Review by valearna — CHANGES_REQUESTED** [review]

File: agr_literature_service/api/crud/topic_entity_tag_crud.py (hunk 1 above)
Comment (valearna) [reply agr_literature_service/api/crud/topic_entity_tag_crud.py:225]: yes, sounds great! Thanks @sweng66

Comment (sweng66) [reply agr_literature_service/api/crud/topic_entity_tag_crud.py:225]: Cool. @valearna

Comment (sweng66) [reply agr_literature_service/api/crud/topic_entity_tag_crud.py]: Fixed

Comment (sweng66) [reply agr_literature_service/api/crud/topic_entity_tag_crud.py]: fixed the parameter names

**Review by valearna — APPROVED** [review]

**Comment by sweng66** [comment]
Thanks @valearna

**Pull request merged**